src/modules/change-email/routes.js

```diff
@@ -0,0 +1,43 @@
+const controller = require('./controller');
+const Joi = require('joi');
+
```

Comment:
I think we will need a few extra items (e.g. userId)  in the payload for several of these routes as mentioned in the controller above.